### PR TITLE
rename the label for the Don't list in the admin panel

### LINKED ISSUE #143: Rename dont label to "don't"
The label of the "dont" field in the `DontBlock` should have an apostrophe "Don't"

image

### PR DESCRIPTION
**Pull request by AdamChrimes**
resolves #143 

Dont is now Don't

Screenshot 2020-04-14 at 22 38 36

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ wagtail_nhsuk_frontend.egg-info
 wagtailnhsukfrontend/static/wagtailnhsukfrontend/css/wagtail-nhsuk-frontend.min.css
 testapp/db.sqlite3
 testapp/media/images
+.vscode
```
Comment (mikemonteith) [inline .gitignore:5]: @AdamChrimes  These ignores are local to your system, not the project.
You can set up your own local gitignore: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

**Comment by mikemonteith** [comment]
I think this can be rebased on master to remove the dependency on #163 ?

**Comment by AdamChrimes** [comment]
I've fixed up this PR @mikemonteith

**Comment by AdamChrimes** [comment]
There was a missing migration for the testapp homepage, I've fixed this.

**Pull request merged**